msu/load.nut

```diff
@@ -29,3 +29,6 @@ includeLoad("hooks");
 includeLoad("msu_mod");
 includeLoad("vanilla_mod");
 includeLoad("test");
+
```

Comment:
Please create a bug report for this on the vanilla forums and add a link to that in a comment here as we do for all other vanilla fixes.

Reply:
https://steamcommunity.com/app/365360/discussions/1/3436829654723733817/